seqr/views/apis/variant_search_api.py

```diff
@@ -366,6 +380,33 @@ def _get_projects_details(projects, user, project_category_guid=None):
     return response
 
 
+def _add_parent_ids(response, projects, family_models, individual_models):
```

Comment:
Although the two `_add_parent_ids` are in two different files,  it is still a little confusing. Maybe you want to eliminate this function and inline the codes.